engine/apps/alerts/tasks/notify_user.py

```diff
@@ -196,7 +196,15 @@ def notify_user_task(
 
         if not stop_escalation:
             if notification_policy.step != UserNotificationPolicy.Step.WAIT:
-                transaction.on_commit(partial(perform_notification.apply_async, (log_record.pk,)))
+
+                def _create_perform_notification_task():
+                    task = perform_notification.apply_async((log_record.pk,))
+                    task_logger.info(
+                        f"Created perform_notification task {task} log_record={log_record.pk} "
+                        f"alert_group={alert_group_pk}"
+                    )
+
+                transaction.on_commit(_create_perform_notification_task)
```

Comment:
Just wondering: can we face with the same issue with not existing objects as when we used transaction.on_commit() with `lambda` instead of `partial`?